administrator/components/com_patchtester/script.php

```diff
@@ -48,6 +51,24 @@ public function __construct()
 	 */
 	public function postflight($type, $parent)
 	{
-		$this->removeFiles();
+		if ($type == 'install')
+		{
+			$language = JFactory::getLanguage();
+			$language->load('com_patchtester', JPATH_ADMINISTRATOR, null, true);
+			$language->load('com_patchtester', JPATH_SITE, null, true);
+			echo '<h1><img src="../media/com_patchtester/images/icon-48-patchtester.png"> ' . Text::_('COM_PATCHTESTER') . '</h1>';
```

Comment:
You can close the PHP tag before this line and just write out the HTML code.

Reply:
Not much of coder myself, just manage to make minor tweaks to code like this that help others.
So I would close the PHP tag on line 58 and then html below?

Reply:
It would look like this:
```
$language->load('com_patchtester', JPATH_SITE, null, true);
?>
<h1><img src="../media/com_patchtester/images/icon-48-patchtester.png"><?php echo Text::_('COM_PATCHTESTER'); ?></h1>
```
After that the other HTML lines. Once the HTML is done, you can start PHP again with `<?php`